src/System.Data.SqlClient/tests/ManualTests/SQL/InstanceNameTest/InstanceNameTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    public static class InstanceNameTest
+    {
+        [CheckConnStrSetupFact]
+        public static void ConnectToSQLWithInstanceNameTest()
+        {
+            string tcpConnStr = DataTestUtility.TcpConnStr;
```

Comment:
Don't need this variable, since you can just pass it to the connection string builder directly.